src/database/bookingHeaderDb.ts

```diff
@@ -0,0 +1,22 @@
+import { dbConnection } from './dbConnection';
+import { BookingHeader } from '../interfaces/BookingHeader';
+
+export const bookingHeaderDb = {
+  async insert(bookingHeader: BookingHeader): Promise<number> {
+    const connection = await dbConnection();
+
+    bookingHeader.BOOKING_HEADER_NO = connection.raw(
+      'CvsBookingHeader.nextVal',
```

Comment:
is CvsBookingHeader a val for a column name in the db?

Reply:
It's a sequence in Oracle. It's a database object that increments with each call to nextVal. It's like an identity column, but can be shared across multiple tables.